server/locales/en.js

```diff
@@ -0,0 +1,159 @@
+      new: {
+        title: 'Registration',
+        form: {
+          firstName: 'First Name',
+          lastName: 'Last Name',
+          email: 'Email',
+          password: 'Password',
+          confirm: 'Register',
+        },
+      },
+      index: {
+        list: {
+          columns: {
+            email: 'Email',
+            firstName: 'First Name',
+            lastName: 'Last Name',
+            action: {
+              name: 'Action',
+              edit: 'Edit',
+              delete: 'Delete',
+            },
+          },
+        },
+      },
+    },
+    session: {
+      signin: {
+        title: 'Sign In',
+        form: {
+          email: 'Email',
+          password: 'Password',
+          confirm: 'Sign In',
+        },
+      },
+    },
+    statuses: {
+      new: {
+        title: 'New Status',
+        form: {
+          name: 'Name',
+          confirm: 'Save',
+        },
+      },
+      edit: {
+        title: 'Edit Status',
+        form: {
+          name: 'Name',
+          confirm: 'Save',
+        },
+      },
+      index: {
+        add: 'New',
+        list: {
+          columns: {
+            name: 'Status Name',
+            action: {
+              name: 'Action',
+              edit: 'Edit',
+              delete: 'Delete',
+            },
+          },
+        },
+      },
+    },
+    labels: {
+      new: {
+        title: 'New Label',
+        form: {
+          name: 'Name',
+          confirm: 'Save',
+        },
+      },
+      index: {
+        add: 'New',
+        list: {
+          columns: {
+            name: 'Label Name',
+            action: {
+              name: 'Action',
+              edit: 'Edit',
+              delete: 'Delete',
+            },
+          },
+        },
+      },
+      edit: {
+        title: 'Edit',
+        form: {
+          name: 'Label Name',
+          confirm: 'Save',
+        },
+      },
+    },
+    tasks: {
+      new: {
+        title: 'New Task',
+        form: {
+          name: 'Name',
+          description: 'Description',
+          status: 'Status',
+          labels: 'Labels',
+          creator: 'Creator',
+          assigned: 'Assigned',
+          confirm: 'Save',
+        },
+      },
+      edit: {
+        title: 'Edit Task',
+        form: {
+          name: 'Name',
+          description: 'Description',
+          status: 'Status',
+          labels: 'Labels',
+          creator: 'Creator',
+          assigned: 'Assigned',
+          confirm: 'Save',
+        },
+      },
+      index: {
+        add: 'New',
+        list: {
+          columns: {
+            name: 'Name',
+            description: 'Description',
+            status: 'Status',
+            labels: 'Labels',
+            creator: 'Creator',
+            assigned: 'Assigned',
+            action: {
+              name: 'Action',
+              edit: 'Edit',
+              delete: 'Delete',
+            },
+          },
+        },
+      },
+    },
+  },
+};
```

Comment:
Probably it would be better to separate views and flashes in here.
Here is the list of what hexlet check expects from buttons:
```json
  "buttons": {
    "save": {
      "name": "Сохранить"
    },
    "edit": {
      "name": "Изменить"
    },
    "delete": {
      "name": "Удалить"
    },
    "signIn": {
      "name": "Войти"
    },
    "signOut": {
      "name": "Выход"
    },
    "create": {
      "name": "Создать"
    },
    "show": {
      "name": "Показать"
    }
  },
```